src/style.css

```diff
@@ -0,0 +1,134 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
+
+body,
+div,
+p {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: #ffff;
+  height: 406px;
+  width: 198px;
+  font-family: "Roboto", "Franklin Gothic Medium", Tahoma, sans-serif;
+  position: absolute;
```

Comment:
you don't need absolute here, it must be static element. Absolute elements is hard to reuse without parent